eventkit_cloud/api/views.py

```diff
@@ -2290,6 +2294,94 @@ def get(self, request, *args, **kwargs):
         return Response(payload, status=status.HTTP_200_OK)
 
 
+class MetricsView(views.APIView):
+    """
+     This view should return a list of metrics detailing the use of the platform
+    """
+
+    @action(detail=False, methods=["get"])
+    def get(self, request, *args, **kwargs):
+        """
+         Args:
+             days: number of days from which to gather information. E.g. if '30' is specified
+             the metrics will be from the last 30 days.
+             area_type: the specific type of areas on which to group download events
+             area_count: the number of 'top areas' to display, i.e. top 5 areas with the most downloads
+             user_group: one or more user characteristics on which to group users.
+             group_count: the number of top user groups to display, i.e. top 10 user groups with the most downloads
```

Comment:
Params listed here, an example query string would be: 
http://host.docker.internal/api/metrics?days=7&area_type=countries&area_count=10&user_group=deptId&user_group=service&org_count=5